developer/src/inst/download.in.mak

```diff
@@ -105,17 +105,16 @@ makeinstaller:
     $(SIGNCODE) /d "Keyman Developer" keymandeveloper-$Version.exe
 
 #
-# Zip the files we distribute as part of the standalone kmcomp distro into release\$Version\kmcomp-$Version.zip
+# Zip the files we distribute as part of the standalone kmc distro into release\$Version\kmcomp-$Version.zip
 #
 
-KMCOMP_ZIP=$(DEVELOPER_ROOT)\release\$Version\kmcomp-$Version.zip
+# TODO: rename this to keyman-developer-cli-$Version.zip
+KMC_ZIP=$(DEVELOPER_ROOT)\release\$Version\kmcomp-$Version.zip
```

Comment:
Should this be changed to `kmc-$Version.zip` ?

Reply:
For now we are keeping it as `kmcomp-$Version.zip` because that impacts the distribution in a bunch of places. Later, we should probably call it `keyman-developer-$Version.zip`, but that can be a separate PR. (It's only a filename for distribution)